packages/cerebral/src/devtools/index.js

```diff
@@ -250,6 +248,9 @@ class Devtools {
       window.dispatchEvent(event)
     }
   }
+  sendError () {
+
```

Comment:
Oh, ops... I started something and forgot to remove it when I found the right implementation :) WIll rebase